spec-dtslint/observables/iif-spec.ts

```diff
@@ -0,0 +1,21 @@
+import { iif, of } from 'rxjs';
+
+it('should accept function as first parameter', () => {
+  // Typescript limitation: cannot infer default value in params
+  // e.g.: const a = iif(() => false, EMPTY, EMPTY) // $ExpectType Observable<never>
+  // However, test below does not infer Observable<never> if no params is passed
+  // although the default value for second and third param is EMPTY observable
+  const a = iif(() => false); // $ExpectType Observable<{}>
```

Comment:
This seems like a bug... we could add a type overload to cover this.

Reply:
The comments should be able to remove when #4494 is merged.